redbot/core/commands/commands.py

```diff
@@ -917,24 +1032,72 @@ async def disabler(ctx: "Context") -> bool:
         return disabler
 
 
-# This is intentionally left out of `__all__` as it is not intended for general use
-class _AlwaysAvailableCommand(Command):
+# The below are intentionally left out of `__all__`
+# as they are not intended for general use
+class _AlwaysAvailableMixin:
```

Comment:
The usage of `_AlwaysAvailableCommand` in `isinstance` check in `[p]command disable` (2 occurrences) needs to be replaced with this mixin (or possibly `_RuleDropper` as it seems to be used for everything else).